.github/scripts/run-tests.sh

```diff
@@ -110,7 +110,8 @@ case $1 in
         make run-binary-fast -C $LOCAL_SIM_DIR $DISABLE_SIM_PREREQ ${mapping[$1]} BINARY=$RISCV/riscv64-unknown-elf/share/riscv-tests/benchmarks/multiply.riscv
         ;;
     chipyard-ibex)
-        run_bmark ${mapping[$1]} #TODO: Find 32-bit test
+        # Ibex cannot run the riscv-tests binaries for some reason
+        # make run-binary-fast -C $LOCAL_SIM_DIR $DISABLE_SIM_PREREQ ${mapping[$1]} BINARY=$RISCV/riscv64-unknown-elf/share/riscv-tests/isa/rv32ui-p-simple
```

Comment:
Do we have any easy way to compile a 32b test? If not, maybe we should just check that Ibex compiles (and not runs)

Reply:
Not at the moment, the default libgloss install is only rv64.

Ibex seems to fail this due to not implementing some necessary CSR :man_shrugging: